src/Npgsql/Internal/Converters/Primitive/DoubleConverter.cs

```diff
@@ -1,39 +1,15 @@
-using System;
-using System.Numerics;
-
 namespace Npgsql.Internal.Converters;
 
-sealed class DoubleConverter<T> : PgBufferedConverter<T>
```

Comment:
Nobody's asked for it so far. We could add it for consistency with the other numeric conversions (especially since we plan to move those out to opt ins), but whatever. Up to you.